src/UnscentedKalmanInversion.jl

```diff
@@ -743,7 +748,10 @@ function update_ensemble_analysis!(
 
     if process.impose_prior
         ug_cov_reg = [ug_cov uu_p_cov]
-        gg_cov_reg = [gg_cov ug_cov'; ug_cov uu_p_cov+process.prior_cov[u_idx, u_idx] / get_Δt(uki)[end]]
+        gg_cov_reg = [
+            gg_cov ug_cov'
+            ug_cov uu_p_cov+process.prior_cov[u_idx, u_idx] / get_Δt(uki)[end]
+        ]
         tmp = ug_cov_reg / gg_cov_reg
```

Comment:
Should we not add the linear solve here too for the unscented methods?

i think `A / B` is like `(B' \ A')'`